bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+. /usr/local/docker/env
+
+if [[ "${1:0:1}" = '-' ]]; then
+  # assume $1 is renovate flag
```

Comment:
Is this fallback new?

Reply:
yes, so you can now run `docker run --rm -it renovate/renovate bash` useful for debugging